auth/src/main/java/org/robojackets/apiary/auth/Authentication.kt

```diff
@@ -0,0 +1,259 @@
+                content = {
+                    ChangeEnvironmentBottomSheetContent(
+                        viewState,
+                        onAppEnvChange,
+                        scaffoldState
+                    )
+                }
+            )
+        }) {
+        Surface(
+            color = MaterialTheme.colors.background,
+            modifier = Modifier.padding(8.dp)
+        ) {
+            Column(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .fillMaxHeight(),
+                horizontalAlignment = Alignment.CenterHorizontally,
+                verticalArrangement = Arrangement.SpaceBetween
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.ic_robobuzz_white_outline),
+                    contentDescription = "RoboJackets logo",
+                    modifier = Modifier
+                        .fillMaxWidth(.45f)
+                        .weight(1.0f)
+                )
+                Column(
+                    modifier = Modifier.weight(.5f),
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                    verticalArrangement = Arrangement.SpaceAround
+                ) {
+
+                    Button(
+                        onClick = {
+                            val authRequest = authManager.getAuthRequest()
+                            launcher.launch(
+                                authManager.authService.getAuthorizationRequestIntent(
+                                    authRequest
+                                )
+                            )
+                        },
+                    ) {
+                        Text("Sign in with MyRoboJackets")
+                    }
+                }
+
+                if (viewState.loginStatus == ERROR) {
+                    AlertDialog(
+                        onDismissRequest = {
+                            viewModel.setLoginStatus(NOT_STARTED)
+                        },
+                        confirmButton = {
+                            TextButton(onClick = { viewModel.setLoginStatus(NOT_STARTED) }) {
+                                Text("Close")
+                            }
+                        },
+                        title = { Text("Login failed") },
+                        text = {
+                            Text("${viewState.loginErrorMessage}\n\nTry logging in again. " +
+                                    "If that does not work, please post in #it-helpdesk in Slack.")
+                        },
+                    )
+                }
+
+                if (BuildConfig.DEBUG) {
+                    TextButton(onClick = {
+                        coroutineScope.launch {
+                            scaffoldState.bottomSheetState.expand()
+                        }
+                    }) {
+                        Text("Change server")
```

Comment:
If/when we allow arbitrary servers, the picker should live in the three dots in the top right that some apps have (I think it's called the Overflow Menu?). The vast majority of people will not need to use it, so it doesn't make sense as part of the main interface, but it would be convenient for developers to test the app against a test (or local!) server without having to build it locally and sideload.

Reply:
Right now there is no top app bar for that.  That could change in the future but I don't see it as an initial priority